Document how to publish a new module version for a branch based module. (#637)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/changelog.mdx

# API Changelog

Keep track of changes to the API for HCP Terraform and Terraform Enterprise.

> ## 2024-05-23

Reviewer: ```suggestion

## 2024-05-23
```

> * Update [Registry Modules API](/terraform/cloud-docs/api-docs/private-registry/modules#create-a-module-version) to detail how to publish a new version for a branch based module.

Reviewer: Style suggestions (we only capitalize product names)
```suggestion
* Update the [registry modules API](/terraform/cloud-docs/api-docs/private-registry/modules#create-a-module-version) for publishing new versions of branch based modules.
```
